test/test_helper.rb

```diff
@@ -112,6 +112,11 @@ def self.disable_database_queries
     end
   end
 
+  def self.enable_url_helpers
```

Comment:
Since we already have one of these in the active mailer class, do you think we could refactor it and extract it into a mixin class that gets `include`d in both?